metricflow-semantics/metricflow_semantics/model/semantics/linkable_element.py

```diff
@@ -98,6 +99,16 @@ class SemanticModelJoinPathElement(SerializableDataclass):
 class LinkableElement(SemanticModelDerivation, SerializableDataclass, ABC):
     """An entity / dimension that may have been joined by entities."""
 
+    properties: Tuple[LinkableElementProperty, ...]
+
+    def __post_init__(self) -> None:  # noqa: D105
+        assert len(self.property_set) == len(self.properties)
```

Comment:
Can we add error messages to these asserts to help devs if they run into this in development?

Reply:
Added.